eng/config/PublishData.json

```
@@ -27,14 +27,23 @@
           "vsBranch": "rel/d16.1",
           "vsMajorVersion": 16
         },
-        "master-vs-deps": {
+        "dev16.1-vs-deps": {
             "nugetKind": ["Shipping", "NonShipping"],
             "version": "3.1.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev16.1", "dev16.1p3" ],
+            "channels": [ "dev16.1" ],
```

I think this should still include p3, although I don't know who uses these channels
```suggestion
            "channels": [ "dev16.1", "dev16.1p3" ],
```